### PR TITLE
refactor(cb2-5690): remove the 2 extra brake force sections

### PR DESCRIPTION
**Pull request by tahayazici**
## Amend PSV Brakes section

- Simplify the handling of form state in the tech summary component.
- Remove the extra 2 brake sections as they are now encompassed within the brake force table.

[CB2-5690](https://dvsa.atlassian.net/browse/CB2-5690)

### CODE REVIEW

**Review by tom-bjss — DISMISSED** [review]

File: src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts [hunk 1]
```diff
@@ -130,22 +128,26 @@ export class TechRecordSummaryComponent implements OnInit {
...
+  handleFormState(event: any): void {
+    this.vehicleTechRecordCalculated = merge(cloneDeep(this.vehicleTechRecordCalculated), event);
 
-    for (const ax of axles) {
-      if (ax.axleNumber === previousAxleRow) {
-        previousAxleRow += 1;
-      } else {
-        return ax.axleNumber! - 1;
-      }
+    if (event.axles && event.axles.length < this.vehicleTechRecordCalculated.axles.length) {
```
Comment (aaron-downing) [inline src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:154]: Do you need the truthy check on `event.axles`, is it not possible to just do:
`if (event.axles?.length < this.vehicleTechRecordCalculated.axles.length)`

Comment (tahayazici) [reply src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:154]: The first `event.axles` check is explicit. It checks whether the event was triggered by the axles section.

Comment (naathanbrown) [reply src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:154]: This is the exact one that I was saying broke so if it works with this I will quit

File: src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts [hunk 2]
```diff
@@ -130,22 +128,48 @@ export class TechRecordSummaryComponent implements OnInit {
     this.store.dispatch(updateEditingTechRecord({ techRecord: this.vehicleTechRecordCalculated }));
   }
 
-  findAxleToRemove(axles: Axle[]): number {
-    let previousAxleRow = 1;
+  handleFormStateOriginal(event: any): void {
```
Comment (naathanbrown) [inline src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts:131]: Why are we leaving the original in place?

Comment (tahayazici) [reply src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts]: My bad, debugging... Now removed 🙂

**Review by naathanbrown — APPROVED** [review]

**Comment by aaron-downing** [comment]
Making backend change to allow `null`/`''` instead of passing undefined

**Review by tom-bjss — APPROVED** [review]

**Pull request merged**